docs-src/concepts/what-is-a-workflow-definition.md

```diff
@@ -23,7 +23,7 @@ We strongly recommend that you write a Workflow Definition in a language that ha
 
 A critical aspect of developing Workflow Definitions is ensuring they exhibit certain deterministic traits – that is, making sure that the same Commands are emitted in the same sequence, whenever a corresponding Workflow Function Execution (instance of the Function Definition) is re-executed.
 
-The execution semantics of a Workflow Execution include the re-execution of a Workflow Function, which is called a [Replay](/concepts/what-is-a-workflow-execution#replays).
+The execution semantics of a Workflow Execution include the re-execution of a Workflow Function, which is called a [Replay](/app-dev-context/replays).
```

Comment:
I recommend changing the link target from `/app-dev-context/replays` (in "Testing") to `#replays` (in this topic). Doing so removes the preview, but the tighter contextual connection is more valuable, IMO.